docs/data/charts/sparkline/generate-sparkline.js

```diff
@@ -0,0 +1,64 @@
+// Run `node generate-sparkline.js > weekly-downloads.json` To update the npm chart demo.
```

Comment:
Why not make this dynamic? Eg on page load? I fear this will be forgotten here 😆

Reply:
> Why not make this dynamic? Eg on page load? I fear this will be forgotten here 😆

Because data does not need to be up to date. I just made this script to get a realistic look and feel (nothing is more realistic than real data O:) )

I'm not fan of a browser API call. first to avoid bothering npm server, and mostly to avoid having a broken page if they modify their API/their server are for download stats are down